### PR TITLE
[pkgmgr] Support "rpk" install

### PR DESCRIPTION
**Pull request by anyj0527**
- When rpk is installed, parse the provided "model_description.json" file
  and register models described in the json.

TODO: Handle when the rpk package is removed or updated.

### CODE REVIEW

File: daemon/includes/pkg-mgr.h [hunk 1]
```diff
@@ -18,17 +18,16 @@
...
-#endif /* __cplusplus */
-
 #if defined(__TIZEN__)
 #include <glib.h>
 #include <stdio.h>
 
 #include "log.h"
 #include "package_manager.h"
 
+#ifdef __cplusplus
```
Comment (jaeyun-jung) [inline daemon/includes/pkg-mgr.h:28]: weird change..

Comment (anyj0527) [reply daemon/includes/pkg-mgr.h]: Updated, PTAL.

**Review by again4you — APPROVED** [review]
Basically, LGTM!

Trivial Question)
Should we consider `PACKAGE_MANAGER_EVENT_TYPE_RES_COPY` event?

**Comment by jaeyun-jung** [comment]
plz rebase this also

**Comment by anyj0527** [comment]
> Basically, LGTM!
> 
> Trivial Question) Should we consider `PACKAGE_MANAGER_EVENT_TYPE_RES_COPY` event?

Well I think we need to when these events could be used, the doc is saying that these are related with resource package. We may ask to the person in charge of package_manager.

**Comment by anyj0527** [comment]
> plz rebase this also

Rebased, Thanks!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**